test/fixtures/tasks.yml

```diff
@@ -0,0 +1,7 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
As we are not using the fixtures for the tests it's better if this file is deleted.